Multisig/Logic/Models/KeyInfo.swift

```diff
@@ -10,13 +10,24 @@ import Foundation
 import CoreData
 import Web3
 
+/// Enum for storing key type in the presistanse store. The order of existing items should not be changed.
+enum KeyType: Int, CaseIterable {
+    case deviceImported
```

Comment:
I recommend adding ` = 0` and `= 1` explicitly so that order can be changed

Reply:
ok